src/components/ADempiere/ContainerOptions/DocumentStatusTag/index.vue

```diff
@@ -0,0 +1,97 @@
+<template>
```

Comment:
This is a generic behavior, applies for tables, fields and reports. You should try use it as field or generic components

Reply:
It is already implemented in the tables, fields and other places where it is to be used.